local-cli/runIOS/runIOS.js

```diff
@@ -65,18 +87,46 @@ function runIOS(argv, config, args) {
   child_process.spawnSync('xcrun', ['simctl', 'launch', 'booted', bundleID], {stdio: 'inherit'});
 }
 
-function matchingSimulator(simulators, simulatorName) {
-  for (let i = simulators.length - 1; i >= 0; i--) {
-    if (simulators[i].name === simulatorName || formattedSimulatorName(simulators[i]) === simulatorName) {
-      return simulators[i];
+function runOnDevice(selectedDevice, scheme, xcodeProject){
+  buildProject(xcodeProject, selectedDevice.udid, scheme);
+  const iosDeployInstallArgs = [
+    '--bundle', 'build/Build/Products/Debug-iphoneos/' + scheme + '.app',
+    '--id' , selectedDevice.udid,
+    '--justlaunch'
+  ];
+  console.log('installing and launching app on connected device...');
+  child_process.spawnSync('ios-deploy', iosDeployInstallArgs, {stdio: 'inherit'});
```

Comment:
What's the developer experience for this. Will people need to remember to also install ios-deploy?